src/metadataMIME.ml

```diff
@@ -0,0 +1,33 @@
+(** Guess the mime-type of a file. *)
+
+let prefixes =
+  [
+    "ID3", "audio/mpeg";
```

Comment:
This is not gonna be reliable..

Reply:
It's going to be a fine enough guess, isn't it?

Reply:
Sure!